doc/Methods and Results - Rough.Rmd

```diff
@@ -0,0 +1,32 @@
+---
+title: "Methods and Results - Rough"
+author: "Farhan Majumder"
+date: "December 4, 2017"
+output: html_document
+---
+
+# Methods 
+
+## Dataset
```

Comment:
Pay attention to how the subheadings are structured in the final assignment description https://uoftcoders.github.io/rcourse/assignment-final.html. You are free to add additional headings, for example to partition your result section further. But the suggested heading "Data description" is more descriptive than just saying "Data set" here.